README.md

```diff
@@ -26,7 +26,22 @@
           "app_category_type": "public.app-category.productivity"
         },
         "linux": {
-          "icon": "resources/linux/<your-project-id>.png"
+          "icon": "resources/linux/<your-project-id>.png",
+          "debian_config": {
+            "section": "Databases",
+            "depends": [
+              "python"
+            ],
+            "suggests": [
+              "libgnome-keyring0"
+            ]
+          },
+          "redhat_config": {
+            "category": "Applications/Databases",
+            "requires": [
+              "libXScrnSaver(x86-64)"
```

Comment:
@imlucas Ah cool that makes sense. At least we have some inkling what the GitHub notifications issue might be going forward, will try to use `#compass-scrum` on Slack more but would be nice to resolve in case we get more external interest in our https://github.com/mongodb-js/ repos.